### DISCUSSION TITLE
add nginx modules geoip2

### DISCUSSION THREAD

**Discussion by ghost** [NginxProxyManager/nginx-proxy-manager#1531] [Ideas]
Hello,
it's possible to geoip2 nginx module?

https://github.com/leev/ngx_http_geoip2_module.

Thanks!

**Comment by chaptergy** [comment]
It is not, but there is a [feature request](https://github.com/jc21/nginx-proxy-manager/issues/46) about that

**Comment by ghost** [comment]
some useful  materials:
https://www.electrosoftcloud.com/en/compile-geoip2-in-openresty-and-how-to-use-it/
